packages/docs/src/pages/index.md

```diff
@@ -9,6 +9,13 @@ It is currently being applied to [HealthCare.gov](https://www.healthcare.gov/).
 
 ## Getting started
 
+<div class="ds-c-alert">
+  <div class="ds-c-alert__body">
+    <h2 class="ds-c-alert__heading">CMS Design System v4 has been released</h2>
+    <p class="ds-c-alert__text">For more information on v4 and how to upgrade, see our <a href="https://design.cms.gov/startup/migrating-v4/">migration guide</a>.</p>
```

Comment:
Would it actually work to use `{{root}}` here like we do later in this file? Or does it not work in HTML sections?

Reply:
Ah good point, i think it might work let me try

Reply:
{{root}} might only work in the context of Markdown

Reply:
I see other examples of it in example html files, but I haven't seen another one in a markdown file. Technically this is HTML inside a markdown file so it should work... I just cant test because the doc building is a bit messed up right now :/

Reply:
Right now that link will at least be valid in production. We can circle back to it later

Reply:
nevermind! i got it, it works!